components/url/lib.rs

```diff
@@ -163,6 +165,16 @@ impl ServoUrl {
 
 impl fmt::Display for ServoUrl {
     fn fmt(&self, formatter: &mut fmt::Formatter) -> fmt::Result {
+        //changes for bug 22485
```

Comment:
Let's remove this comment; we can find this information out from the VCS history of this commit.